.github/CODEOWNERS

```diff
@@ -136,12 +160,49 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
 src/applications/mhv/landing-page @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-cto-health-products
 src/applications/avs @department-of-veterans-affairs/va-cto-health-products
 
-# VA.gov Product Teams - Forms
+# Dependents Benefits
+src/applications/static-pages/dependency-verification @department-of-veterans-affairs/benefits-ce-dependents
+src/applications/static-pages/view-modify-dependent @department-of-veterans-affairs/benefits-ce-dependents
+src/applications/static-pages/view-modify-dependents @department-of-veterans-affairs/benefits-ce-dependents
 
-src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
+# Flagship Mobile Reviewers
+src/applications/static-pages/subscription-creators @department-of-veterans-affairs/flagship-mobile-reviewers
+src/applications/static-pages/widget-creators @department-of-veterans-affairs/flagship-mobile-reviewers
 
-# Other
+# VA CTO Health Products
+
+src/applications/static-pages/covid-vaccine-updates-cta @department-of-veterans-affairs/va-cto-health-products
+
+# VAOS
 
 src/applications/vaos @department-of-veterans-affairs/vfs-vaos-fe
+src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page  @department-of-veterans-affairs/vfs-vaos-fe @department-of-veterans-affairs/vsa-public-websites-frontend
+
+# Other
+
 src/applications/dhp-connected-devices @department-of-veterans-affairs/digital-health-platform
 src/applications/virtual-agent @department-of-veterans-affairs/orchid
+src/applications/pre-need @department-of-veterans-affairs/bah-mbs-selfserv
+
+# TODO: who owns these?
+src/applications/ask-a-question
+src/applications/ask-va
+src/applications/e-folders
+src/applications/messages
+src/applications/mock-sip-form
+src/applications/post-911-gib-status
+src/applications/terms-of-use
+src/applications/third-party-app-directory
+src/applications/verify
+src/applications/veteran-id-card
```

Comment:
I think you can assign this one to `benefits-team-1`, that's one of my teams applications

Reply:
Updated! Thank you